api/v1alpha1/application.go

```diff
@@ -58,6 +60,12 @@ type Replicas struct {
 	TargetCpuUtilization uint `json:"targetCpuUtilization,omitempty"`
 }
 
+type Strategy struct {
+	// +kubebuilder:validation:Enum=RollingUpdate;Recreate
+	// +kubebuilder:default=RollingUpdate
+	Type string `json:"type,omitempty"`
+}
+
```

Comment:
Noen grunn til å introdusere en egen struct bare for denne stringen? Har vi planer om mer rollout strategy options down the line?

Reply:
Tenkte det ja, begynte med dette nå så NRL kan få redeploya databasen sin så kan vi legge inn options for `RollingUpdate` og slikt her etter hvert